### PR TITLE
github-action: upload chaosctl to s3

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?
 

Upload `chaosctl` binary to s3. Users can download it directly using `curl -sSL https://mirrors.chaos-mesh.org/${version}/chaosctl`

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/upload_latest_files.yml [hunk 1]
```diff
@@ -12,6 +12,12 @@ jobs:
     name: Upload
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Go 1.14
+        uses: action/setup-go@v1
```
Comment (g1eny0ung) [inline .github/workflows/upload_latest_files.yml:16]: I think the v2 version can be used, which can be more stable.

**Review by yeya24 — APPROVED** [review]
Looks good. Might be better to address @g1eny0ung's comment.

**Review by AsterNighT — APPROVED** [review]

File: .github/workflows/upload_latest_files.yml (hunk 1 above)
Comment (cwen0) [reply .github/workflows/upload_latest_files.yml:16]: Got it!

**Comment by STRRL** [comment]
Maybe we need binaries for both darwin and linux?

**Comment by AsterNighT** [comment]
> Maybe we need binaries for win, darwin and linux?

Based upon my experience on installing chaosmesh on windows, I believe it is not that necessary to have a windows binary :facepalm:

File: .github/workflows/upload_latest_files.yml [hunk 2]
```diff
@@ -12,6 +12,12 @@ jobs:
     name: Upload
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Go 1.15
```
Comment (AsterNighT) [inline .github/workflows/upload_latest_files.yml:15]: Go 1.15 will run into #1618, please wait for #1599

**Comment by YangKeao** [comment]
> > Maybe we need binaries for win, darwin and linux?
> 
> Based upon my experience on installing chaosmesh on windows, I believe it is not that necessary to have a windows binary facepalm

The cluster manager may use the windows, with the Chaos Mesh installed on his remote Linux cluster.

**Comment by STRRL** [comment]
Hi guys how about merging the PR at first then make up the binaries of the rest platform in another PR?

**Comment by g1eny0ung** [comment]
> Hi guys how about merging the PR at first then make up the binaries of the rest platform in another PR?

I agree.

**Pull request merged**